packages/sdk/source/network.models.ts

```diff
@@ -12,6 +12,8 @@ export type WalletPermission = "read" | "write";
 
 export type TransactionType =
 	| "delegateRegistration"
+	| "usernameRegistration"
+	| "usernameResignation"
 	| "delegateResignation"
```

Comment:
```suggestion
	| "delegateRegistration"
	| "delegateResignation"
	| "usernameRegistration"
	| "usernameResignation"
```

let's keep the types together

Reply:
done